Makefile

```diff
@@ -87,7 +87,12 @@ gencatalog:
 	rm site/content/en/guides/consumer/function/catalog/*/_index.md
 	(cd site/content/en/guides/consumer/function/catalog/catalog && npm run gen-docs)
 
+site-generate:
+	go run ./scripts/generate_site_sidebar > site/sidebar.md
+	(cd site && find . -iname "00.md" -execdir mv {} README.md \; && sed -i s/00.md//g sidebar.md)
```

Comment:
Can this be encapsulated in the go program? Currently, how content is arranged sprinkled across Go, Bash, and Javascript.

Reply:
I don't think it would make much sense putting the renaming of `00.md` files to `README.md` in the go program. Ideally, that line would only be executed before deployment; perhaps that normalization should be added specifically as as a `run` line in the `firebase-hosting-merge.yml` action.